go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/mna/pigeon
+
+go 1.12
```

Comment:
Can you please change this also to `go 1.11`.

Reply:
I regenerated go.mod with go 1.11 and it just dropped that line